src/java.xml/share/classes/com/sun/org/apache/bcel/internal/classfile/AnnotationEntry.java

```diff
@@ -1,6 +1,5 @@
 /*
- * reserved comment block
- * DO NOT REMOVE OR ALTER!
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
IANAL, but I think the original year needs to be here, as apparently it has been in the repository.

Also, I see other files, e.g., `ArrayElementValue.java` has the same header ("reserved comment block"). Should they also have Oracle copyright?

Reply:
Oracle copyright is added when we make changes to the upstream source. In this case, I replaced the newly introduced reference to Apache Commons Lang with a Utils method, thus the copyright year 2025 since we haven't made changes to the upstream source until now. Also added is the LastModified tag as required by the license. Note the tag was not there until now.

Reply:
If Apache's bcel constantly introduce new dependencies, monkey patches like these are no better than migrating the whole xsltc to use ClassFile API for bytecode generation, in my opinion.